modules/machineset-modifying.adoc

```diff
@@ -72,26 +122,49 @@ NAME                        PHASE     TYPE         REGION      ZONE         AGE
 +
 [source,terminal]
 ----
-$ oc annotate machine/<machine_name_original_1> \
+tag::CAPI[]
+$ oc annotate machine.cluster.x-k8s.io/<machine_name_original_1> \
+  -n openshift-cluster-api \
+  cluster.x-k8s.io/delete-machine="true"
+#CAPI values are guesses, need validation#
```

Comment:
Tested, this is correct, just to be consistent with other commands, change all `machine.cluster.x-k8s.io` to `machines.cluster.x-k8s.io`,` machineset.cluster.x-k8s.io` to `machinesets.cluster.x-k8s.io` ?